CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 ### HEAD
 * Update to Bootstrap 3.3.4
 * Fix `gulp --production` race condition ([#1398](https://github.com/roots/sage/issues/1398))
+* Removes pleeease dependency in favor of vanilla gulp-autoprefixer and
+    gulp-minify-css
```

Comment:
Remove pleeease dependency in favor of vanilla gulp-autoprefixer and gulp-minify-css ([#1402](https://github.com/roots/sage/issues/1402))